.circleci/install_hera.sh

```diff
@@ -0,0 +1,40 @@
+#! /bin/bash
+#------------------------------------------------------------------------------
+# Bash script that install hera ewasm evmc vm. Script will be removed once
+# hera is part of the docker images.
+#
+# The documentation for solidity is hosted at:
+#
+#     https://solidity.readthedocs.org
+#
+# ------------------------------------------------------------------------------
+# This file is part of solidity.
+#
+# solidity is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# solidity is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with solidity.  If not, see <http://www.gnu.org/licenses/>
+#
+# (c) 2016-2019 solidity contributors.
+# ------------------------------------------------------------------------------
+
+apt-get update
+apt-get upgrade -y
```

Comment:
Is this really needed? It makes the image less predictable.

Reply:
yeah, it is needed, at least for now.. it looks like that there is a problem with the current gcc that is installed within the docker images. The installed gcc had problems compiling hera.. with an error like that one:
```
/usr/lib/gcc/x86_64-linux-gnu/9/include-fixed/bits/statx.h:38:25: error: missing binary operator before token "("
   38 | #if __glibc_has_include ("__linux__/stat.h")
      |  
```
After upgrading gcc the compilation was successful. We would need to recreate the docker images and update the packages to the latest version.